ci: TestKprobeSkb fails on bpf-next (6.5-rc1)

https://github.com/cilium/tetragon/pull/1247

```
time="2023-07-19T07:36:04Z" level=info msg="BTF discovery: default kernel btf file found" btf-file=/sys/kernel/btf/vmlinux
=== RUN   TestKprobeSkb
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="Disabling Cilium API"
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="BTF discovery: default kernel btf file found" btf-file=/sys/kernel/btf/vmlinux
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="Starting process manager" enableCilium=false enableK8s=false enableProcessCred=true enableProcessNs=true
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="Starting JSON exporter"
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=warning msg="Kprobe spec pre-validation failed, but will continue with loading" error="type (sk_buff) of argument 1 does not match spec type (skb)\n" sensor=gkp-sensor-1
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="Added generic kprobe sensor: /home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_kprobe_v61.o -> __cgroup_bpf_run_filter_skb" flags=none
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="BTF file: using metadata file" metadata=/sys/kernel/btf/vmlinux
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="Loading sensor" name=__base__
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="Loading kernel version 6.5.0"
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="tetragon, map loaded." map=execve_map path=/sys/fs/bpf/testSensorTracing/execve_map sensor=__base__
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="tetragon, map loaded." map=execve_map_stats path=/sys/fs/bpf/testSensorTracing/execve_map_stats sensor=__base__
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="tetragon, map loaded." map=execve_calls path=/sys/fs/bpf/testSensorTracing/execve_calls sensor=__base__
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="tetragon, map loaded." map=names_map path=/sys/fs/bpf/testSensorTracing/names_map sensor=__base__
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="tetragon, map loaded." map=tcpmon_map path=/sys/fs/bpf/testSensorTracing/tcpmon_map sensor=__base__
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="tetragon, map loaded." map=tg_conf_map path=/sys/fs/bpf/testSensorTracing/tg_conf_map sensor=__base__
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="Loading BPF program" Program=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_exit.o Type=kprobe
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="Loading BPF program" Program=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_fork.o Type=kprobe
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="Loading registered BPF probe" Program=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_execve_event_v61.o Type=execve
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="Read ProcFS /proc appended 15/126 entries"
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=warning msg="Procfs execve event pods/ identifier error" error="open /proc/0/cgroup: no such file or directory"
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="Loaded BPF maps and events for sensor successfully" sensor=__base__
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="BTF file: using metadata file" metadata=/sys/kernel/btf/vmlinux
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="Loading sensor" name=datagram
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="Loading kernel version 6.5.0"
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="tetragon, map loaded." map=fdinstall_map path=/sys/fs/bpf/testSensorTracing/gkp-sensor-1-fdinstall_map sensor=datagram
    logcapture.go:25: time="2023-07-19T07:36:04Z" level=info msg="tetragon, map loaded." map=config_map path=/sys/fs/bpf/testSensorTracing/gkp-sensor-1-gkp-0-config_map sensor=datagram
    logcapture.go:25: time="2023-07-19T07:36:05Z" level=info msg="tetragon, map loaded." map=kprobe_calls path=/sys/fs/bpf/testSensorTracing/gkp-sensor-1-gkp-0-kp_calls sensor=datagram
    logcapture.go:25: time="2023-07-19T07:36:05Z" level=info msg="tetragon, map loaded." map=filter_map path=/sys/fs/bpf/testSensorTracing/gkp-sensor-1-gkp-0-filter_map sensor=datagram
    logcapture.go:25: time="2023-07-19T07:36:05Z" level=info msg="tetragon, map loaded." map=argfilter_maps path=/sys/fs/bpf/testSensorTracing/gkp-sensor-1-gkp-0-argfilter_maps sensor=datagram
    logcapture.go:25: time="2023-07-19T07:36:05Z" level=info msg="tetragon, map loaded." map=retprobe_map path=/sys/fs/bpf/testSensorTracing/gkp-sensor-1-gkp-0-retprobe_map sensor=datagram
    logcapture.go:25: time="2023-07-19T07:36:05Z" level=info msg="tetragon, map loaded." map=process_call_heap path=/sys/fs/bpf/testSensorTracing/gkp-sensor-1-gkp-0-process_call_heap sensor=datagram
    logcapture.go:25: time="2023-07-19T07:36:05Z" level=info msg="tetragon, map loaded." map=sel_names_map path=/sys/fs/bpf/testSensorTracing/gkp-sensor-1-gkp-0-sel_names_map sensor=datagram
    logcapture.go:25: time="2023-07-19T07:36:05Z" level=info msg="tetragon, map loaded." map=socktrack_map path=/sys/fs/bpf/testSensorTracing/gkp-sensor-1-socktrack_map sensor=datagram
    logcapture.go:25: time="2023-07-19T07:36:05Z" level=info msg="Loading registered BPF probe" Program=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_kprobe_v61.o Type=generic_kprobe
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="Loaded generic kprobe program: /home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_kprobe_v61.o -> __cgroup_bpf_run_filter_skb"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="Loaded BPF maps and events for sensor successfully" sensor=datagram
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="Successfully detected bpftool path" bpftool=/usr/sbin/bpftool
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="Starting metrics server" addr="localhost:2112"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="Perf ring buffer size (bytes)" percpu=69632 total=139264
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="Listening for events..."
    jsonchecker.go:179: jsonTestCheck: opening: /tmp/tetragon.gotest.TestKprobeSkb.2830065509.json
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0001 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0002 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0003 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0004 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0005 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0006 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0007 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0008 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0009 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0010 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0011 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0012 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0013 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0014 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0015 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:06Z" level=info msg="jsonTestCheck/line:0016 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    jsonchecker.go:138: JsonCheck (retry=0) took 16.339762ms
    jsonchecker.go:162: JsonCheck (retry=1) failed: JsonEOF: failed to match after 16 events: err:UnorderedEventChecker: 1/1 checks remain. Retrying after 3s
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0001 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0002 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0003 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0004 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0005 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0006 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0007 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0008 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0009 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0010 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0011 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0012 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0013 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0014 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0015 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking event with 0/1 total matched"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="UnorderedEventChecker: checking pending 1/1: failure: ProcessKprobeChecker/datagram-checker: *tetragon.ProcessExec is not a ProcessKprobe event"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="jsonTestCheck/line:0016 event:PROCESS_EXEC => no match: UnorderedEventChecker: all 1 check(s) failed, continuing"
    jsonchecker.go:138: JsonCheck (retry=1) took 8.038616ms
    kprobe_test.go:3600: 
        	Error Trace:	/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/pkg/sensors/tracing/kprobe_test.go:3600
        	Error:      	Received unexpected error:
        	            	JsonTestCheck failed in retry cnt=1 and there were no new events from previous try: JsonEOF: failed to match after 16 events: err:UnorderedEventChecker: 1/1 checks remain
        	Test:       	TestKprobeSkb
    filenames.go:47: keeping export file for TestKprobeSkb (/tmp/tetragon.gotest.TestKprobeSkb.2830065509.json)
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="Listening for events completed." error="context canceled"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="saving init info"
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=info msg="retrieving lib directory" libDir=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_alignchecker.ll
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_cgroup_mkdir.ll
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_cgroup_release.ll
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_cgroup_rmdir.ll
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_execve_event.ll
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_execve_event_v53.ll
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_execve_event_v61.ll
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_exit.ll
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_fork.ll
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_kprobe.ll
    logcapture.go:25: time="2023-07-19T07:36:09Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_kprobe_v53.ll
    logcapture.go:25: time="2023-07-19T07:36:10Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_kprobe_v61.ll
    logcapture.go:25: time="2023-07-19T07:36:10Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_retkprobe.ll
    logcapture.go:25: time="2023-07-19T07:36:10Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_retkprobe_v53.ll
    logcapture.go:25: time="2023-07-19T07:36:10Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_retkprobe_v61.ll
    logcapture.go:25: time="2023-07-19T07:36:10Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_tracepoint.ll
    logcapture.go:25: time="2023-07-19T07:36:10Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_tracepoint_v53.ll
    logcapture.go:25: time="2023-07-19T07:36:10Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_tracepoint_v61.ll
    logcapture.go:25: time="2023-07-19T07:36:10Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_uprobe.ll
    logcapture.go:25: time="2023-07-19T07:36:10Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_uprobe_v53.ll
    logcapture.go:25: time="2023-07-19T07:36:10Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_generic_uprobe_v61.ll
    logcapture.go:25: time="2023-07-19T07:36:10Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_globals.ll
    logcapture.go:25: time="2023-07-19T07:36:10Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_loader.ll
    logcapture.go:25: time="2023-07-19T07:36:10Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_lseek.ll
    logcapture.go:25: time="2023-07-19T07:36:10Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_multi_kprobe_v53.ll
    logcapture.go:25: time="2023-07-19T07:36:11Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_multi_kprobe_v61.ll
    logcapture.go:25: time="2023-07-19T07:36:11Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_multi_retkprobe_v53.ll
    logcapture.go:25: time="2023-07-19T07:36:11Z" level=warning msg="not an object file, ignoring" path=/home/runner/work/tetragon/tetragon/go/src/github.com/cilium/tetragon/bpf/objs/bpf_multi_retkprobe_v61.ll
    logcapture.go:25: time="2023-07-19T07:36:11Z" level=info msg="btf file added" btfFname=/sys/kernel/btf/vmlinux
    logcapture.go:25: time="2023-07-19T07:36:11Z" level=info msg="tetragon log file added" exportFname=/tmp/tetragon.gotest.TestKprobeSkb.2830065509.json
    logcapture.go:25: time="2023-07-19T07:36:11Z" level=info msg="contacting metrics server" metricsAddr="http://localhost:2112/metrics"
    logcapture.go:25: time="2023-07-19T07:36:11Z" level=warning msg="failed to contact metrics server" err="Get \"http://localhost:2112/metrics\": dial tcp: lookup localhost on 168.63.129.16:53: dial udp 168.63.129.16:53: connect: network is unreachable" metricsAddr="http://localhost:2112/metrics"
    logcapture.go:25: time="2023-07-19T07:36:11Z" level=info msg="executed command" cmd=/usr/bin/dmesg dstFname=dmesg.out ret=0
    logcapture.go:25: time="2023-07-19T07:36:11Z" level=info msg="executed command" cmd="/usr/sbin/tc filter show dev lo ingress" dstFname=tc-info.lo.ingress ret=0
    logcapture.go:25: time="2023-07-19T07:36:11Z" level=info msg="executed command" cmd="/usr/sbin/tc filter show dev lo egress" dstFname=tc-info.lo.egress ret=0
    logcapture.go:25: time="2023-07-19T07:36:11Z" level=info msg="executed command" cmd="/usr/sbin/tc filter show dev sit0 ingress" dstFname=tc-info.sit0.ingress ret=0
    logcapture.go:25: time="2023-07-19T07:36:11Z" level=info msg="executed command" cmd="/usr/sbin/tc filter show dev sit0 egress" dstFname=tc-info.sit0.egress ret=0
    logcapture.go:25: time="2023-07-19T07:36:11Z" level=info msg="executed command" cmd="/usr/sbin/bpftool map show -j" dstFname=bpftool-maps.json ret=0
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="executed command" cmd="/usr/sbin/bpftool prog show -j" dstFname=bpftool-progs.json ret=0
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="executed command" cmd="/usr/sbin/bpftool cgroup tree -j" dstFname=bpftool-cgroups.json ret=0
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=warning msg="failed to open policyfilter map" error="no such file or directory"
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=warning msg="failed to connect to : failed to build resolver: passthrough: received empty target in Build()"
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="Dumped bugtool info" file=/tmp/tetragon-bugtool.tar.gz test=TestKprobeSkb
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="BPF prog was unloaded" label=kprobe/do_task_dead pin=event_exit
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="BPF prog was unloaded" label=kprobe/wake_up_new_task pin=kprobe_pid_clear
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="BPF prog was unloaded" label=tracepoint/sys_execve pin=event_execve
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="BPF prog was unloaded" label=kprobe/generic_kprobe pin=gkp-sensor-1-gkp-0-__cgroup_bpf_run_filter_skb_prog
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=execve_map pin=execve_map
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=execve_map_stats pin=execve_map_stats
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=execve_calls pin=execve_calls
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=names_map pin=names_map
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=tcpmon_map pin=tcpmon_map
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=tg_conf_map pin=tg_conf_map
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=fdinstall_map pin=gkp-sensor-1-fdinstall_map
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=config_map pin=gkp-sensor-1-gkp-0-config_map
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=kprobe_calls pin=gkp-sensor-1-gkp-0-kp_calls
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=filter_map pin=gkp-sensor-1-gkp-0-filter_map
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=argfilter_maps pin=gkp-sensor-1-gkp-0-argfilter_maps
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=retprobe_map pin=gkp-sensor-1-gkp-0-retprobe_map
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=process_call_heap pin=gkp-sensor-1-gkp-0-process_call_heap
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=sel_names_map pin=gkp-sensor-1-gkp-0-sel_names_map
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="map was unloaded" map=socktrack_map pin=gkp-sensor-1-socktrack_map
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="BPF events statistics: 0 received, 0% events loss"
    logcapture.go:25: time="2023-07-19T07:36:12Z" level=info msg="Observer events statistics" errors=0 filterDrop=0 filterPass=0 lost=0 received=0
--- FAIL: TestKprobeSkb (8.08s)
FAIL
```

--- a/pkg/sensors/tracing/kprobe_test.go
+++ b/pkg/sensors/tracing/kprobe_test.go
@@ -4143,7 +4143,7 @@ metadata:
   name: "datagram"
 spec:
   kprobes:
-  - call: "__cgroup_bpf_run_filter_skb"
+  - call: "ip_send_skb"
     syscall: false
     args:
     - index: 1
@@ -4170,7 +4170,7 @@ metadata:
   name: "datagram"
 spec:
   kprobes:
-  - call: "__cgroup_bpf_run_filter_skb"
+  - call: "ip_send_skb"
     syscall: false
     args:
     - index: 1
@@ -4207,7 +4207,7 @@ spec:
 	res.LookupIP(context.Background(), "ip4", "ebpf.io")
 
 	kpChecker := ec.NewProcessKprobeChecker("datagram-checker").
-		WithFunctionName(sm.Full("__cgroup_bpf_run_filter_skb")).
+		WithFunctionName(sm.Full("ip_send_skb")).
 		WithArgs(ec.NewKprobeArgumentListMatcher().
 			WithOperator(lc.Ordered).
 			WithValues(

--- a/pkg/vmtests/skip.go
+++ b/pkg/vmtests/skip.go
@@ -14,7 +14,6 @@ type skipRule struct {
 
 // We should probably have this in a json file, but for now we keep it here
 var rules = []skipRule{
-	skipRule{TestNameRe: "pkg.sensors.tracing.TestKprobeSkb", KernelRe: "bpf-next"},
 	skipRule{TestNameRe: "pkg.sensors.tracing.TestLoader", KernelRe: "bpf-next"},
 	skipRule{TestNameRe: "pkg.tracepoint.TestTracepointLoadFormat", KernelRe: "bpf-next"},
 }

